### ISSUE TITLE
Option to use game cover as background image if none is available

### ISSUE THREAD

**Issue by Kvaell** [JosefNemec/Playnite#1018]
Hello,

I wanted to know if it would be possible by default that playnite automatically apply the game cover also as the background cover for each game?

Regards

**Comment by DennoCoil** [comment]
Pretty sure it already does this, but it really depends on the Store the game is coming from.  For example, Steam and GOG do it, but everything else doesn't support it.

So the real idea I think should be, where does the background cover come from?  Lets say a random screenshot, but a screenshot could be anything or the wrong game.  Maybe from IGDB's screenshot section of a game's metadata?  You'd still run into the same problem.

What Calibre (open source ebook manager, highly recommended) does is it grabs the first result from a selected number of sites that's used to download metadata.

https://farm8.staticflickr.com/7830/46195023585_802fbcca63_o.png
https://farm8.staticflickr.com/7896/40144089493_ba00787964_o.png

It's much more advanced than Playnite, but this could be a feature in the future.  It would still require API's from a half dozen sites.  (Some of those places in the 2nd picture come from a forum and GitHub repos.)

**Comment by Kvaell** [comment]
Thanks for the answer! 

Having game screenshot seems nice but as you said, it would requires much work. 

For me the same picture for the game cover and background cover is enough, so it would only require a script that automatically take the picture saved for the game cover and apply it on the background cover.

**Comment by DennoCoil** [comment]
So maybe a toggle function where in the absence of a background image, a zoomed in copy of the cover art would be best.  It would be nice to have more functionality in the future, but it's better than just a blank wall in any case.

It already scales the background image using the top left of the program as an anchor point, but it's hard coded to zoom in to scale to the shortest length.  The background image below is zoomed in to the width of the image because that's the shortest.

![2019-02-16 21_48_30-](https://user-images.githubusercontent.com/21137176/52907743-b0a88380-3235-11e9-9d8a-807c9b4791bd.png)
(Note: I used the PAL cover I found on a search.  IGDB didn't download a background image.)
